blazor/dialog/predefined-dialogs.md

```diff
@@ -0,0 +1,338 @@
+platform: Blazor
+control: Dialog
+documentation: ug
+---
+
+# Predefined dialogs in the Blazor Dialog component
+
+The dialog component is used to render the `Alert`, `Confirm`, and `Prompt` dialogs with minimal code. The alert, confirm, and prompt dialogs are shown using [DialogServices](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html).
+
+## Configuration
+
+### Blazor Server App
+
+* For **.NET 6** app, open the **~/Program.cs** file and import Syncfusion.Blazor.Popups.
+
+* For **.NET 5 and .NET 3.X** app, open the **~/Startup.cs** file and import Syncfusion.Blazor.Popups.
+
+{% tabs %}
+{% highlight c# tabtitle=".NET 6 (~/Program.cs)" hl_lines="3 10" %}
+
+@using Syncfusion.Blazor.Popups
+
+// Add services to the container.
+builder.Services.AddRazorPages();
+builder.Services.AddServerSideBlazor();
+builder.Services.AddScoped<SfDialogService>();
+builder.Services.AddSyncfusionBlazor(options => { options.IgnoreScriptIsolation = true; });
+
+var app = builder.Build();
+....
+
+{% endhighlight %}
+
+{% highlight c# tabtitle=".NET 5 and .NET 3.X (~/Startup.cs)" hl_lines="1 12" %}
+
+@using Syncfusion.Blazor.Popups
+
+namespace BlazorApplication
+{
+    public class Startup
+    {
+        ...
+        public void ConfigureServices(IServiceCollection services)
+        {
+         	services.AddRazorPages();
+    		services.AddServerSideBlazor();
+    		services.AddScoped<SfDialogService>();
+            services.AddSyncfusionBlazor(options => { options.IgnoreScriptIsolation = true; });
+    		services.AddServerSideBlazor().AddCircuitOptions(options => { options.DetailedErrors = true; });
+    		services.AddServerSideBlazor().AddHubOptions(o =>
+        }
+        ...
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Blazor WebAssembly App
+
+Open **~/Program.cs** file and register the Syncfusion Blazor Service in the client web app.
+
+{% tabs %}
+{% highlight C# tabtitle=".NET 6 (~/Program.cs)" hl_lines="3 11" %}
+
+@using Syncfusion.Blazor.Popups
+
+// Add services to the container.
+
+builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+builder.Services.AddScoped<SfDialogService>();
+builder.Services.AddSyncfusionBlazor(options => { options.IgnoreScriptIsolation = true; });
+await builder.Build().RunAsync();
+....
+
+{% endhighlight %}
+
+{% highlight c# tabtitle=".NET 5 and .NET 3.X (~/Program.cs)" hl_lines="1 10" %}
+
+@using Syncfusion.Blazor.Popups
+
+namespace WebApplication1
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            ....
+            builder.Services.AddSyncfusionBlazor(options => { options.IgnoreScriptIsolation = true; });
+            builder.Services.AddScoped<SfDialogService>();
+            await builder.Build().RunAsync();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+open  **~/_MainLayout.razor** file and add SfDialogProvider
```

Comment:
casing problem. Was this reviewed by content team? Also, it will come under WASM -> Is it fine?

Reply:
Hi @syncsiva 
Was this resolved?